realm/realm-library/src/androidTest/java/io/realm/RealmObjectTests.java

```diff
@@ -2125,9 +2125,17 @@ public void setter_string_long_values() {
     @Test
     public void setter_nonLatinFieldName() {
         // Reproduces https://github.com/realm/realm-java/pull/5346
-        realm.beginTransaction();
-        NonLatinFieldNames obj = realm.createObject(NonLatinFieldNames.class);
-        obj.setΔέλτα(0);
-        realm.commitTransaction();
+        RealmConfiguration config = configFactory.createConfigurationBuilder().name("greekWTF").schema(NonLatinFieldNames.class).build();
+        Realm realm = Realm.getInstance(config);
+        try {
+            realm.beginTransaction();
+            RealmObjectSchema schema = realm.getSchema().get(NonLatinFieldNames.class.getSimpleName());
+            schema.getFieldType("델타");
```

Comment:
If you call `schema.getFieldType()` on a different field, then the setter will crash?? O_O

Reply:
No, it crashes there. I just kept the rest for further test.